api/desecapi/tests/testdomains.py

```diff
@@ -11,22 +11,22 @@
 
 class UnauthenticatedDomainTests(APITestCase):
     def testExpectUnauthorizedOnGet(self):
-        url = reverse('domain-list')
+        url = reverse('v1:domain-list')
```

Comment:
The version string is duplicated here many many times. How about importing a wrapped version of this, e.g. `from desec.test.utils import reverse`? We could have this function use some test-global state (as we have for the test user settings) to store the version number, and then just call the native `reverse()` with the first argument augmented.

Reply:
I considered and dismissed that idea. Each test is written for a specific version of the API, reusing tests is complicated and may be of limited use. Because we cannot decide today how to reuse tests, as we do not know how the future version may look like, it is fair to update all tests to their specific version. Admittedly, it looks hideous in the commit diff, but it's not a lot of work with search and replace if it needs to be changed later. Again, today, I do not see a scenario where it is useful to change all tests to another API version.